styles/badge.css

```diff
@@ -0,0 +1,95 @@
+@import "./global.css";
+
+:root {
+  /* status */
+  --online-status: var(--success-color);
+  --offline-status: grey;
+  --dnd-status: var(--danger-color);
+  --badge-dot-width: 0.75rem;
+  --badge-dot-height: 0.75rem;
+  --badge-num-width: 1.5rem;
+  --badge-num-height: 1.5rem;
+}
+
+/* badge class */
+.badge {
+  position: relative;
+}
+
+/* badge position */
+
+.badge-dot-size {
+  width: var(--badge-dot-width);
+  height: var(--badge-dot-height);
+}
+
+.badge-pos-bottom-right {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  border-radius: 50%;
+  border: 2px solid #ffff;
```

Comment:
Use color variable

Reply:
thanks, Aiman for reminding me about this.